helm/dagster/schema/schema/charts/dagster/subschema/compute_log_manager.py

```diff
@@ -16,43 +16,44 @@ class ComputeLogManagerType(str, Enum):
 
 
 class AzureBlobComputeLogManager(BaseModel):
+    secretKey: Optional[StringSource] = None
+    defaultAzureCredential: Optional[dict] = None
+    localDir: Optional[StringSource] = None
+    prefix: Optional[StringSource] = None
+    uploadInterval: Optional[int] = None
+
```

Comment:
why all this thrash in this PR? Will our users have to do this as well? Or is this purely as pydantic v2 thing?

Reply:
This is due to [a change in how Pydantic treats required & nullable values](https://docs.pydantic.dev/latest/blog/pydantic-v2/#required-vs-nullable-cleanup) - because these fields were `Optional` but didn't have a default value, Pydantic v2 treats them as "required", and will error if a value is not provided. (Pydantic v1 would just default to `None`)